### PR TITLE
Portfolio dev

### PR DESCRIPTION
**Pull request by coding-sue-true**
All main structure and content is done, but it still looks like a pre historic webpage.
Until Friday will make it responsive and modern.

### CODE REVIEW

File: portfolio/about.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <body>
+    <!-- ------------------------ NAV BAR CONTENT ------------------ -->
+    <nav>
+      <h1><span class="header_name"><a href="index.html"> Soraia Carmo </a></h1>
+      <ul>
+        <span class="nav_items">
+          <li><a href="about.html">About</a></li>
+          <li><a href="projects.html">Projects</a></li>
+          <li><a href="contact.html">Contact</a></li>
+      </span>
```
Comment (JSChilds) [inline portfolio/about.html]: Lines 18 and 19 need to be indented.

File: portfolio/contact.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  <body>
+    <!-- ------------------------ NAV BAR CONTENT ------------------ -->
+    <nav>
+      <h1><span class="header_name"><a href="index.html"> Soraia Carmo </a></h1>
+      <ul>
+        <span class="nav_items">
+          <li><a href="about.html">About</a></li>
+          <li><a href="projects.html">Projects</a></li>
+          <li><a href="contact.html">Contact</a></li>
+      </span>
```
Comment (JSChilds) [inline portfolio/contact.html:18]: Needs to be indented

File: portfolio/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+footer {
+  /*background-color: red;*/
+  border: 1px solid black;
+  text-align: center;
+}
+
+nav h1 {
+  display: inline-block;
+}
+
```
Comment (JSChilds) [inline portfolio/styles/style.css:86]: Try to avoid leaving in commented-out code.

**Review by JSChilds — APPROVED** [review]
Keep an eye on indentation, but otherwise very good.

File: portfolio/contact.html (hunk 2 above)
Comment (coding-sue-true) [reply portfolio/contact.html:18]: thanks joe!

**Pull request merged**